met-web/src/App.tsx

```diff
@@ -59,7 +59,7 @@ const App = () => {
 
         try {
             const tenant = await getTenant(basename);
-
+            document.title = tenant.name;
```

Comment:
better to use React Helmet

Reply:
this got pushed to the wrong branch removed